core/src/main/java/org/mobilitydata/gtfsvalidator/notice/Notice.java

```diff
@@ -54,7 +54,7 @@ public boolean equals(Object other) {
     if (other instanceof Notice) {
       Notice otherNotice = (Notice) other;
       return context.equals(otherNotice.context)
-          && getCode().equals(otherNotice.getCode())
+          && getClass().getSimpleName().equals(otherNotice.getClass().getSimpleName())
```

Comment:
getClass().equals(otherNotice.getClass())
&& severityLevel.equals(otherNotice.severityLevel)
&& context.equals(otherNotice.context)

That is optimized for speed.

Reply:
Modified in https://github.com/MobilityData/gtfs-validator/pull/789/commits/c0732911e4b070236fb67dd0ded790698de764e8. Thanks :)

Reply:
It still was not modified in a way that I suggested but you can submit it as-is.